src/main/kotlin/Navigation.kt

```diff
@@ -0,0 +1,13 @@
+import java.util.Scanner
+
+class Navigator(private val scanner: Scanner) { // объявляем класс для навигации по приложению
+     private val archives = mutableListOf<Archive>() // создадим новый список, состоящий из пустых архивов
+
+    fun start(){ //отсюда начнём работу приложения
```

Comment:
```suggestion
    fun start() { //отсюда начнём работу приложения
```
Перед коммитом стоит приводить файлы к правильному стилю, так как код будут смотреть другие разработчики. В IDEA это легко делать путём вызова горячей комбинации клавиш (она зависит от ОС), её можно посмотреть в меню Code → Reformat Code